src/pages/registration/FileList.tsx

```diff
@@ -166,6 +166,12 @@ export const FileList = ({ title, files, uppy, remove, baseFieldName }: FileList
                                 {license.link}
                               </Link>
                             )}
+                            {license.i18nKey && (
+                              <Trans
+                                i18nKey={`licenses.additional_info.${license.i18nKey}` as any}
```

Comment:
ikke spesielt fornøyd med å bruke `any` her, men fant ikke noen bedre løsning. Det fungerer uten, men da får man en linting error.

Reply:
Hvis man skal sende en i18nKey bør man også sørge for at det faktisk er en gydlig nøkkel med å gi den typen `ParseKeys` i stedet for en vilkårlig streng.

Noen grunn til å bruke interpolering med å sende bare en liten del av nøkkelen i stedet for hele her? Da må man passe på at man setter riktig del av nøkkelen her, og at alle konsumenter også oppgir riktig del av nøkkelen. Eneste måten å finne ut på blir vel å sjekke manuelt overalt 🤔